docs/getting-started-guides/photon-controller.md

```diff
@@ -0,0 +1,232 @@
+---
+---
+
+The example below creates a Kubernetes cluster using VMware's Photon
+Controller. The cluster will have one Kubernetes master and three
+Kubernetes nodes.
+
+* TOC
+{:toc}
+
+### Prerequisites
+
+1. You need administrator access to a [VMware Photon
+Controller](http://vmware.github.io/photon-controller/)
+deployment. (Administrator access is only required for the initial
+setup: the actual creation of the cluster can be done by anyone.)
+
+2. The [Photon Controller CLI](https://github.com/vmware/photon-controller-cli)
+needs to be installed on the machine on which you'll be running kube-up. If you
+have go installed, this can be easily installed with:
+
+```shell
+go get github.com/vmware/photon-controller-cli/photon
+```
+
+3. `mkisofs` needs to be installed. The installation process creates a
+CD-ROM ISO image to bootstrap the VMs with cloud-init. If you are on a
+Mac, you can install this with [brew](http://brew.sh/):
+
+```shell
+brew install cdrtools
+```
+
+4. Several common tools need to be installed: `ssh`, `scp`, `openssl`
+
+5. You should have an ssh public key installed. This will be used to
+give you access to the VM's user account, `kube`.
+
+6. Get or build a [binary release](/docs/getting-started-guides/binary_release)
+
+### Download VM Image
+
+Download a prebuilt Debian 8.2 VMDK that we'll use as a base image:
+
+```shell
+curl --remote-name-all https://s3.amazonaws.com/photon-platform/artifacts/OS/debian/debian-8.2.vmdk
+```
+
+This is a base Debian 8.2 image with the addition of:
+
+* openssh-server
+* open-vm-tools
+* cloud-init
+
+### Configure Photon Controller:
+
+In order to deploy Kubernetes, you need to configure Photon Controller
+with:
+
+* A tenant, with associated resource ticket
+* A project within that tenant
+* VM and disk flavors, to describe the VM characteristics
+* An image: we'll use the one above
+
+When you do this, you'll need to configure the
+`cluster/photon-controller/config-common.sh` file with the names of
+the tenant, project, flavors, and image.
+
+If you prefer, you can  use the provided `cluster/photon-controller/setup-prereqs.sh`
```

Comment:
The name of the script is wrong.  Should be:

>
cluster/photon-controller/setup-prereq.sh